packages/jumpcloud/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.15.1"
```

```suggestion
- version: "1.16.0"
```
Enhancements bump minor.